app/.server/parse-markdown.js

```diff
@@ -1,34 +1,37 @@
 import fs from 'node:fs/promises';
 import matter from 'gray-matter';
 import { Marked } from 'marked';
-import markedShiki from 'marked-shiki';
-import { getHighlighter } from 'shiki';
+import { createHighlighter } from 'shiki';
```

Comment:
`getHighlighter` was deprecated in favour of `createHighlighter` in a recent shiki version